dockerfiles/platform-core/Dockerfile

```diff
@@ -39,6 +39,7 @@ RUN apt-get install -y \
   libapr1 \
   python \ 
   psmisc \
+  zip \
```

Comment:
I may be wrong - the structure of the debug-related Dockerfiles is a bit confusing - but I think you also need it in platform-core-debug

Reply:
Right. Add zip to platform-core-debug/Dockerfile